flink-core/src/test/java/org/apache/flink/configuration/RestOptionsTest.java

```diff
@@ -46,7 +46,7 @@ public void testBindAddressFirstDeprecatedKey() {
 	public void testBindAddressSecondDeprecatedKey() {
 		final Configuration configuration = new Configuration();
 		final String expectedAddress = "foobar";
-		configuration.setString("jobmanager.web.address", expectedAddress);
```

Comment:
this specifically tests that the deprecated key remains functional and must be revert.

Reply:
Sorry, I was careless. It's been recovered.